proto/ethermint/sgx/v1/query.proto

```diff
@@ -0,0 +1,396 @@
+message AccessTuple {
+  // Original data type: common.Address
+  bytes address = 1;
+  // Original data type: []common.Hash
+  repeated bytes storage_keys    = 2;
+
+}
+
+message StateDBPrepareRequest {
+  // handler_id is the unique request ID that is given by the SGX binary.
+  uint64 handler_id = 1;
+  // Original data type: common.Address
+  bytes sender         = 2;
+  // Original data type: common.Address
+  bytes coinbase       = 3;
+  // Original data type: common.Address
+  bytes dest           = 4;
+  Rules rules         = 5;
+  repeated AccessTuple access_list = 6;
+}
+
+message StateDBPrepareResponse {
+}
+
+message StateDBGetRefundRequest {
+  // handler_id is the unique request ID that is given by the SGX binary.
+  uint64 handler_id = 1;
+}
+
+message StateDBGetRefundResponse {
+  uint64 refund = 1;
+}
+
+message StateDBGetLogsRequest {
+  // handler_id is the unique request ID that is given by the SGX binary.
+  uint64 handler_id = 1;
+}
+
+message StateDBGetLogsResponse {
+  repeated EthLog log = 1; // []*ethtypes.Log
+}
+
+message Message {
+  // Original type: *common.Address
+	bytes to          = 1;
+  // Original type: common.Address
+	bytes from        = 2;
+	uint64 nonce      = 3;
+  // *big.Int
+	uint64 value      = 4;
+	uint64 gas_limit  = 5;    
+  // Original type: *big.Int
+	uint64 gas_price  = 6;
+  // Original type: *big.Int
+	uint64 gas_fee_cap = 7;
+  // Original type: *big.Int
+	uint64 gas_tip_cap = 8;
+	bytes data        = 9;
+  // Original types: AccessList
+  repeated AccessTuple access_list  = 10 [(gogoproto.nullable) = false];
+  // Original type:  *big.Int
+	uint64 blob_gas_fee_cap           = 11;
+	// Original type: []common.Hash
+  repeated bytes blob_hashes        = 12;
+	// When SkipAccountChecks is true, the message nonce is not checked against the
+	// account nonce in state. It also disables checking that the sender is an EOA.
+	// This field will be set to true for operations like RPC eth_call.
+	bool skip_account_checks          = 13;
+}
+
+message PrepareTxEVMConfig {
+	// ChainConfig is the EVM chain configuration in JSON format. Since the
+	// underlying params.ChainConfig struct contains pointer fields, they are
+	// not serializable over RPC with gob. Instead, the JSON representation is
+	// used.
+	ChainConfig chain_config_json = 1[(gogoproto.nullable) = false];
+	// Fields from EVMConfig
+	// Original type: common.Address
+	bytes coin_base            = 2;
+	// Original type: *big.Int
+	uint64 base_fee            = 3;
+	// Original type: statedb.TxConfig
+	TxConfig tx_config         = 4[(gogoproto.nullable) = false];
+	bool debug_trace           = 5;
+	// Fields from EVMConfig.FeeMarketParams struct
+	bool no_base_fee            = 6;
+	// Fields from EVMConfig.Params struct
+	string evm_denom           = 7;
+	repeated int64 extra_eips  = 8;
+	// *rpctypes.StateOverride : original type
+	bytes overrides = 9;
```

Comment:
Can you add a clearer comment on this field, i.e. why we use `bytes` and what these bytes represent

Reply:
@amaury1093 done.